docs/grid_model/network_subnetwork.md

```diff
@@ -847,8 +847,7 @@ A VSC converter station is made with switching devices that can be turned both o
 ```{warning}
 **The detailed DC model was introduced in iIDM v1.14 and is currently in beta.**
 
-Future releases will enhance this model with support for DC topology processing and serialization/deserialization.  
-These improvements will introduce **breaking changes** and **will not be backward compatible**.
+Future iIDM v1.15 will add support for DC equipment serialization/deserialization.
```

Comment:
I think the right form is `IIDM`. `iIDM` ihas a very few occurences (in the code and the doc) compared to the first one.